.github/workflows/build-external.yml

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+#
+# SPDX-License-Identifier: MIT
+
+name: Build against external 3rdparty
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os:
+          - macos-latest
+        shared:
+          - ON
+          - OFF
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          brew tap KDAB/tap
+          brew install fmt spdlog KDBindings
```

Comment:
btw. do you know why we don't need to install ninja? Something pulls it as dependency?

Reply:
Hmm, https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md doesn't list it as included directly. Something must be pulling it. But well, if it works, let's leave it like it is. If it breaks, we'll add ninja explicitly

Reply:
ah, ninja is listed as a build dependency in the homebrew formulas